packages/hydra-cli/src/generate/VariantsRenderer.ts

```diff
@@ -10,6 +10,16 @@ export class VariantsRenderer extends AbstractRenderer {
     super(model, context)
   }
 
+  withImports(): { imports: string[] } {
+    const moduleImports = new Set<string>()
+    this.model.variants.forEach((v) => {
+      if (v.fields.find((f) => f.type === 'BigInt')) {
+        moduleImports.add(`import BN from 'bn.js'\n`)
```

Comment:
Do we have this additional check for `BigInt` in other templates? Just wonder why we had this issue only for variant types.

Reply:
We have it in the model template https://github.com/Joystream/hydra/blob/7d08d3635e08ee61029beaf831c1fe80fc10be4a/packages/hydra-cli/src/templates/entities/model.ts.mst#L20

Reply:
Ok, don't really like this duplication of the imports logic, but let's settle with a quick fix for now.